docs/uptime-guide/overview.asciidoc

```diff
@@ -1,11 +1,56 @@
 [role="xpack"]
 [[uptime-overview]]
-== Overview
+== Elastic Uptime overview
 
-The Elastic Uptime solution built on the Elastic Stack.
-It allows you to monitor the availability and response time of software services and applications in real time.
+The Elastic Uptime solution is built on the Elastic Stack.
+Elastic Uptime allows you to monitor the availability and response times of applications and services in real time and to detect problems before they affect users.
+
+Elastic Uptime is based on {heartbeat-ref}/index.html[Heartbeat], an open source uptime monitor.
+Elastic Uptime is transparent, flexible, and developer-friendly.
+It can help you to understand uptime and response time characteristics for your services and applications.
+It can be deployed both inside and outside your organization's network, so you can analyze problems from multiple vantage points.
+
+Elastic Uptime consists of these components: *Heartbeat*, *Elasticsearch* and *Kibana*.
+
+[float]
+=== Heartbeat
+
+{heartbeat-ref}/index.html[Heartbeat] is an open source data shipper that performs uptime monitoring.
+Heartbeat allows you to collect monitoring data from your target applications and services, and ship it to Elasticsearch.
+
+[float]
+=== Elasticsearch
```

Comment:
It feels weird for us to define our core products like ES and Kibana in our uptime docs. I would think we could assume this knowledge already.

Reply:
I don't think these are definitions as such. Elasticsearch and Kibana are components in the diagram, so I think they need a sentence of explanation at least to give some content. This may help some readers see the bigger picture.

Currently, the different Observability products have slightly different descriptions (and diagrams) here. We're looking to standardise them, so the descriptions of Elasticsearch and Kibana may become more succinct shortly.

Reply:
It sounds like there have already been discussions about maintainability and consistency here. The chief concern in my original comment was verbosity and redundancy.

Most people using uptime are going to have some experience with the Elastic stack already. It does them a disservice to have docs that repeat what they already know, as the ratio of new info to stuff they already know is worsened.

Is there a docs-level discussion where we've decided to explain what ES/Kibana are everywhere they're mentioned in every product? If so, I'd be curious to read that thread.

Reply:
It is always a fine line between telling users too much, and not telling them enough, but if I had to make a choice I'd err on the side of telling them too much. So long as the explanation is short, concise and clearly signposted, readers who already know it will probably not even notice it, and if they do, they can ignore it, whereas readers who don't know it will find it enormously helpful, and may flounder without it.

Here, the content is the introductory topic, which is a good place to set the scene and name the parts. Elasticsearch is shown as a component in the architecture diagram. And in this particular case, I think a short description of each component in the diagram and the part it plays is essential. I don't think it's redundant at all.

> Is there a docs-level discussion where we've decided to explain what ES/Kibana are everywhere they're mentioned in every product? If so, I'd be curious to read that thread.

No, not yet, but the words I have used here are fairly standard (with minor variations) across the APM, Infrastructure and SEIM introductions, for example, and probably elsewhere. There are minor inconsistencies, and I may yet raise an issue for consistency here. If I do, I'll tag you.